pkg/globalnet/controllers/ipam/globalnet_egress.go

```diff
@@ -0,0 +1,32 @@
+package ipam
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/coreos/go-iptables/iptables"
+	"k8s.io/klog"
+
+	"github.com/submariner-io/submariner/pkg/routeagent/controllers/route"
+)
+
+func (i *Controller) updateEgressRulesForPod(podIP, globalIP string, addOrDelete Operation) error {
+	ipt, err := iptables.New()
+	if err != nil {
+		return fmt.Errorf("error initializing iptables: %v", err)
+	}
+
+	ruleSpec := []string{"-p", "all", "-s", podIP, "-j", "SNAT", "--to", globalIP}
+	if addOrDelete == AddRules {
```

Comment:
wouldn't a switch be preferable? (same for other instances)

Reply:
perhaps even separate the add and the delete into two methods

Reply:
I suggest to use a `bool` for `addOrDelete` since the other `Operation` enums don't apply here.

Reply:
Ack, addOrDelete modified to bool